themes/edx.org/lms/templates/header.html

```diff
@@ -44,18 +44,113 @@
 % endif
 </%block>
 
-% if uses_pattern_library:
+% if uses_bootstrap:
+<header class="navigation-container header-global ${"slim" if course and not disable_courseware_header else ""}" aria-label="Main" role="banner">
+% elif uses_pattern_library:
 <header class="header-global ${"slim" if course and not disable_courseware_header else ""}" aria-label="Main" role="banner">
 % else:
 <header class="${"header-global slim" if course and not disable_courseware_header else "header-global-new"}" aria-label="Main" role="banner">
 % endif
-  <div class="${"rwd" if responsive else ""} wrapper-header nav-container">
 
+% if uses_bootstrap:
+<nav class="navbar navbar-toggleable-md navbar-light" aria-label="${_('Main')}">
```

Comment:
I'm a little concerned by all this new code. Is there any way to share more of this with the default theme? It seems risky as this new logic won't be tested until we switch pages over to Bootstrap. Maybe that's okay as we can find out issues at that point.